### PR TITLE
Add CSI-PowerFlex v2.9 support to csm-operator

### PR DESCRIPTION
**Pull request by VamsiSiddu-7**
# Description
Added CSI-Powerflex v2.9.0 support and removed v2.6.0 config.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/1012 |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Installed the csi-powerflex driver using csm-operator and the installation is successful 
- [x] Ran cert-csi with supported protocols the cert-csi passed with 100%

### CODE REVIEW

File: tests/config/driverconfig/powerflex/v2.9.0/controller.yaml
Comment (jooseppi-luna) [inline tests/config/driverconfig/powerflex/v2.9.0/controller.yaml:1]: can we copy the v2.9.0 changes into this file? Same comment for the other files in this folder

File: tests/shared/common.go [hunk 1]
```diff
@@ -27,10 +27,10 @@ import (
 
 // ConfigVersions used for all unit tests
 const (
-	PFlexConfigVersion       string = "v2.8.0"
```
Comment (jooseppi-luna) [inline tests/shared/common.go:30]: I would only change the PFlex version, I think changing the others is causing the unit tests to fail. ConfigVersion, UpgradeConfigVersion, and JumpUpgradeConfigVersion are used by PScale tests, not PFlex tests.

**Comment by jooseppi-luna** [comment]
Also, would you be able to run pflex e2e tests against this change? I can help with that if needed

File: tests/config/driverconfig/powerflex/v2.9.0/controller.yaml
Comment (VamsiSiddu-7) [reply tests/config/driverconfig/powerflex/v2.9.0/controller.yaml:1]: done

File: tests/shared/common.go (hunk 1 above)
Comment (VamsiSiddu-7) [reply tests/shared/common.go:30]: We need to change these also ConfigVersion, UpgradeConfigVersion, and JumpUpgradeConfigVersion. I verfied powerflex unit tests also depend on the above variables.  The unit tests are passing now

**Comment by VamsiSiddu-7** [comment]
@jooseppi-luna  I have ran e2e test for csi-powerflex driver. I am attaching the screenshot here.

**Comment by VamsiSiddu-7** [comment]
@jooseppi-luna  I have addressed your review comments. Please review it again thanks.

**Review by jooseppi-luna — DISMISSED** [review]

**Review by suryagupta4 — APPROVED** [review]

File: operatorconfig/driverconfig/powerflex/v2.9.0/csidriver.yaml [hunk 2]
```diff
@@ -6,6 +6,7 @@ spec:
     fsGroupPolicy: ReadWriteOnceWithFSType
     attachRequired: true
     podInfoOnMount: true
+    storageCapacity: false
```
Comment (nitesh3108) [inline operatorconfig/driverconfig/powerflex/v2.9.0/csidriver.yaml:9]: This flag must be consistent with helm

**Review by nitesh3108 — APPROVED** [review]

**Review by chimanjain — APPROVED** [review]
LGTM

**Pull request merged**